CommandLine/testing/Platform/TestHarness-X11.cpp

```diff
@@ -174,10 +174,10 @@ int build_game(const string &game, const TestConfig &tc, const string &out) {
   // Invoke the compiler via emake
   using TC = TestConfig;
   string emake_cmd = "./emake";
-  string workdir = (tc.workdir.empty() ? std::string(env_workdir) : tc.workdir);
-  if (!workdir.empty()) workdir = "--workdir=" + workdir;
-  string codegen = (tc.codegen.empty() ? std::string(env_codegen) : tc.codegen);
-  if (!codegen.empty()) codegen = "--codegen=" + codegen;
+  tc.workdir = (tc.workdir.empty() ? std::string(env_workdir) : tc.workdir);
```

Comment:
So here's where you're modifying it. Why does this help anyone? Why does the caller need to know this?

Reply:
I would pass by copy to the function that needs to modify it, then.

Reply:
Alright, I now pass a copy of the TestConfig to `build_game(...)`: f5243a72190a77f019c0aef061668c168f942d2b